ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -396,7 +396,27 @@ tests:
         not @fips'
       TAG_VERSION: '@4.12'
       TEST_ADDITIONAL: Hypershift
-      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;HyperShiftGUEST&
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~NonHyperShiftHOST&;~Serial&;~Disruptive&
+      TEST_TIMEOUT: "30"
+    test:
+    - ref: cucushift-helper-service-build-info
+    - ref: cucushift-pre
+    - ref: openshift-extended-test
+    - ref: cucushift-e2e
+    - ref: openshift-extended-web-tests
+    - ref: cucushift-e2e-report
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: e2e-aws-ipi-ovn-hypershift-guest-p2-destructive
```

Comment:
destructive tests should be p3 (*/28)

Reply:
It is used to push/help ocp component team to debug failed cases in the hypershift hosted cluster. At the beginning, we want to trigger it every week to make debug available.  When it is stable, will change it to p3.  Is that OK ?

Reply:
You can rebuild the job at any time as you wish, which can be used for debug.

Reply:
It is for component team debugging failures every week with this run automatically. It's a little too slow for the debugging process every month.  Anyway, let's find a best way to provide teams to improve their cases in HyperShift.

Reply:
Leave it to you to decide as you are the feature owner.